src/schema/schema_util.c

```diff
@@ -110,6 +110,33 @@ __wt_schema_session_release(WT_SESSION_IMPL *session, WT_SESSION_IMPL *int_sessi
     return (0);
 }
 
+/*
+ * __str_name_check --
+ *     Internal function to disallow any use of the WiredTiger name space. Can be called directly or
+ *     after skipping the URI prefix.
+ */
+static int
+__str_name_check(WT_SESSION_IMPL *session, const char *name, bool skip_wt)
+{
+
+    if (!skip_wt && WT_PREFIX_MATCH(name, "WiredTiger"))
+        WT_RET_MSG(session, EINVAL,
+          "%s: the \"WiredTiger\" name space may not be "
+          "used by applications",
+          name);
+
+    /*
+     * Disallow JSON quoting characters -- the config string parsing code supports quoted strings,
+     * but there's no good reason to use them in names and we're not going to do the testing.
+     */
+    if (strpbrk(name, "{},:[]\\\"'") != NULL)
+        WT_RET_MSG(session, EINVAL,
+          "%s: WiredTiger objects should not include grouping "
+          "characters in their names",
+          name);
+    return (0);
```

Comment:
Might as well join the `WT_RET_MSG` lines?

Reply:
clang-format did that.